runtime/common/src/routing.rs

```diff
@@ -57,10 +57,11 @@ impl<Routers> MessageSender for RouterDispatcher<Routers>
 where
 	Routers: pallet_axelar_router::Config,
 {
+	type Message = Vec<u8>;
```

Comment:
I think the current flow is:
LP gateway serializes message -> LP forwarder deserializes and serializes again -> RouterDispatcher sends the serialized message bytes

Wouldn't it make more sense to have it like:

LP gateway sends the actual LP message -> LP forwarder processes it and sends an LP message -> RouterDispatcher receives an LP message and serializes it when sending

WDYT?

Reply:
> I think the current flow is: LP gateway serializes message -> LP forwarder deserializes and serializes again -> RouterDispatcher sends the serialized message bytes [...]

For serialization, I removed the gateway serialization because that is redundant as you have pointed out.

The forwarder pallet is at least necessary to wrap an LP message into forwarding, in case a forwarder config exists for the given router id. However, it doesn't have to do the actual (de-)serialization. I am more in favor of @lemunozm 's [above proposal](https://github.com/centrifuge/centrifuge-chain/pull/1971#issuecomment-2294360217) of moving the (de-)serialization to a runtime configured type. WDYT?

Reply:
I somehow missed that, nevermind this then.